extensions/amp-iframe/1.0/amp-iframe.js

```diff
@@ -31,6 +33,35 @@ class AmpIframe extends BaseElement {
     );
     return super.isLayoutSupported(layout);
   }
+
+  /** @override */
+  init() {
+    return dict({
+      'onLoadCallback': () => {
+        const placeholder = this.getPlaceholder();
+        if (placeholder) {
+          this.togglePlaceholder(false);
+        } else {
```

Comment:
nit: short-circuit for depth.
```suggestion
        if (placeholder) {
          this.togglePlaceholder(false);
          return;
        }
```